runtime/doc/lsp.txt

```diff
@@ -2168,6 +2168,37 @@ is_enabled({bufnr})                      *vim.lsp.document_color.is_enabled()*
 ==============================================================================
 Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
 
+apply_action({action}, {opts})             *vim.lsp.inlay_hint.apply_action()*
+    For supported LSP servers, apply one of the following actions provided by
```

Comment:
```suggestion
    Apply one of the following actions provided by
```
It is already implied that this will use language servers that support the feature.